### DISCUSSION TITLE
Error Version - vcpkg

### DISCUSSION THREAD

**Discussion by Oboisti-07** [microsoft/vcpkg#40686] [Q&A]
Debian 12 - arm64

```
developer@dev:~/.vcpkg/vcpkg$ vcpkg --version
**vcpkg package management program version 2999-12-31-unknownhash**

See LICENSE.txt for license information.
```

Is correct ?

**Comment by Osyotr** [comment]
It means that the version has not been built by vcpkg team.

**Reply by autoantwort** [reply]
Hm, I would still consider this a bug because it was build with `bootstrap-vcpkg.sh` (at least i guess)
